### ISSUE TITLE
sign of the entropy

### ISSUE THREAD

**Issue by WeiXiongUST** [MrSyee/pg-is-all-you-need#37]
In the implementation of A2C, the code is

        policy_loss += self.entropy_weight * -log_prob  # entropy maximization

But I think since we are maximizing the entropy, in the loss, we shall minimize - H(p), so I guess the negative sign should be removed? I tried out this version and it significantly outperforms the current version.

**Comment by John87654** [comment]
yes. i thought the same.
